to-do-list-backend/src/api/express.ts

```diff
@@ -54,12 +54,36 @@ export function createServer(dependencies: Dependencies): Express {
         resp.send({});
     })
 
-    app.get('/api/tasks', (_, resp: Response) => {
+    app.get('/api/task', (_, resp: Response) => {
```

Comment:
it's better to keep the existing tasks name, because all tasks are listed when this endpoint is called

Reply:
done